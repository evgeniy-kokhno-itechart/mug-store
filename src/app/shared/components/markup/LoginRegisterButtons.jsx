import React from 'react';
import { Link } from 'react-router-dom';

const LoginRegisterButtons = () => (
  <React.Fragment>
    <Link to="/login" className="nav navbar-text mr-3 d-inline clickable">
        Login
    </Link>
    <Link to="/register" className="nav navbar-text d-inline clickable">
        Register
    </Link>
  </React.Fragment>
);

export default LoginRegisterButtons;
