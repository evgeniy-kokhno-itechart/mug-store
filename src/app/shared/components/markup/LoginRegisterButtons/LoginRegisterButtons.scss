@import "Variables";

.login-btn {
    margin-right: 0.5rem;
    &:hover {
        text-decoration: none;
        color: $color-white;
    }
}

.register-btn {
    margin-left: 0.5rem;
    &:hover {
        text-decoration: none;
        color: $color-white;
    }
}