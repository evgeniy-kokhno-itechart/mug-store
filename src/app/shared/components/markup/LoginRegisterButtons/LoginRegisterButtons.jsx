import React from 'react';
import { Link } from 'react-router-dom';
import './LoginRegisterButtons.scss';

const LoginRegisterButtons = () => (
  <React.Fragment>
    <Link to="/login" className="login-btn text--gray">
        Login
    </Link>
    <Link to="/register" className="register-btn text--gray">
        Register
    </Link>
  </React.Fragment>
);

export default LoginRegisterButtons;
