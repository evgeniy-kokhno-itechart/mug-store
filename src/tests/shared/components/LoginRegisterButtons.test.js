/* eslint-disable no-undef */
import React from 'react';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import LoginRegisterButtons from '../../../app/shared/components/markup/LoginRegisterButtons/LoginRegisterButtons';

configure({ adapter: new Adapter() });

describe('<LoginRegisterButtons />', () => {
  let LoginRegisterButtonsWrapper;

  beforeEach(() => {
    LoginRegisterButtonsWrapper = shallow(<LoginRegisterButtons />);
  });

  it('renders properly with default props', () => {
    expect(LoginRegisterButtonsWrapper).toMatchSnapshot();
  });
});
