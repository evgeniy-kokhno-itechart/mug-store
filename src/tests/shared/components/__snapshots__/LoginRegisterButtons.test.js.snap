// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginRegisterButtons /> renders properly with default props 1`] = `
<Fragment>
  <Link
    className="nav navbar-text mr-3 d-inline clickable"
    to="/login"
  >
    Login
  </Link>
  <Link
    className="nav navbar-text d-inline clickable"
    to="/register"
  >
    Register
  </Link>
</Fragment>
`;
